src/components/asset-amount-input/asset-amount-input.tsx

```diff
@@ -2,6 +2,9 @@ import { BigNumber } from 'bignumber.js';
 import React, { FC, memo, useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { Text, TextInput, TouchableOpacity, View } from 'react-native';
 
+import { AnalyticsEventCategory } from 'src/utils/analytics/analytics-event.enum';
+import { useAnalytics } from 'src/utils/analytics/use-analytics.hook';
+
 import { emptyFn } from '../../config/general';
```

Comment:
It would be better not to mix `src` and relative imports of files from the same levels